build.gradle

```diff
@@ -157,9 +157,20 @@ configure(projectsWithFlags('java')) {
         testImplementation 'org.hamcrest:hamcrest-library'
     }
 
-    // Target Java 8.
-    tasks.withType(JavaCompile) {
-        options.compilerArgs += ['--release', '8']
+    tasks.withType(JavaCompile) { task ->
+
+        if (task.path.startsWith(':examples:')) {
+            // Target Java 11 for examples.
+            options.compilerArgs += ['--release', '11']
```

Comment:
`options.release = 11` or similar should work nowadays

Reply:
Thanks, let me try that. 😉